instances/k8smaster/scripts/kubelet.service

```diff
@@ -8,6 +8,7 @@ ExecStart=/usr/bin/kubelet \
   --kubeconfig=http://127.0.0.1:8080 \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --hostname-override=__FQDN_HOSTNAME__ \
+  --cloud-provider=external \
```

Comment:
Should also set `--provider-id` here so that we're not relying on the CCM to resolve the node name to OCID.

Reply:
OK I looked at the docs for the cloud controller and it said that I didn't need this for 1.7.6+

Reply:
I'm going to leave this a seperate MR.

Reply:
We apparently need to move the default back to 1.7.4. So, I'll add the `provider-id` as well to my changes.